lib/trento.pm

```diff
@@ -518,6 +519,44 @@ sub k8s_logs {
             }
         }
     }
+
+    trento_support();
+}
+
+=head3 trento_support
+
+Call trento-support.sh and dump_scenario_from_k8.sh
+and upload the logs
+=cut
+
+sub trento_support {
+    my $machine_ip = get_trento_ip();
+
+    return unless ($machine_ip);
+
+    my $log_dir = 'remote_logs/';
+    enter_cmd "mkdir $log_dir";
+    my $scp_cmd = sprintf 'scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i %s %s@%s',
+      SSH_KEY, VM_USER, $machine_ip;
+    my $cmd = join(' ',
+        'sudo',
+        'KUBECONFIG=/etc/rancher/k3s/k3s.yaml', 'PATH=${PATH}:/home/' . VM_USER . '/bin/',
+        './trento-support.sh',
+        '--output', 'file-tgz',
+        '--collect', 'all');
+    script_run(az_vm_ssh_cmd($cmd, $machine_ip));
+    script_run("$scp_cmd:'*.tar.gz' $log_dir");
+    my @files = split("\n", script_output('ls -1 ' . $log_dir . '*.tar.gz | cat'));
+    foreach my $this_file (@files) {
+        upload_logs($this_file, failok => 1);
+    }
+
+    script_run(az_vm_ssh_cmd('./dump_scenario_from_k8.sh -n goofy', $machine_ip));
+    script_run("$scp_cmd:'scenarios/goofy/*.json' $log_dir");
+    @files = split("\n", script_output('ls -1 ' . $log_dir . '*.json | cat'));
+    foreach my $this_file (@files) {
+        upload_logs($this_file, failok => 1);
+    }
```

Comment:
This `for` block is exactly the same as the one above. I would vote to not have it repeated.

Since you're already using a list with `@files` you can do:
```
my @files = split("\n", script_output('ls -1 ' . $log_dir . '*.tar.gz'));
push @files, split("\n", script_output('ls -1 ' . $log_dir . '*.json'));
foreach my $this_file (@files) {
    upload_logs($this_file, failok => 1);
}
```
or better yet:
```
foreach my $this_file (split("\n", script_output("ls -1 $log_dir *.tar.gz *.json")))
    upload_logs($this_file, failok => 1);
}
```
Fewer calls to the API are always better.